packages/localize-tools/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 0.3.7
+
+### Patch Changes
+
+- [#2113](https://github.com/lit/lit/pull/2113) [`5b2f3642`](https://github.com/lit/lit/commit/5b2f3642ff91931b5b01f8bdd2ed98aba24f1047) - Dependency upgrades including TypeScript 4.4.2
+
+* [#2060](https://github.com/lit/lit/pull/2060) [`dddbe0c7`](https://github.com/lit/lit/commit/dddbe0c7627a7c1f750da69c3200d373155b1d74) - Update TypeScript
```

Comment:
Same comment (This entry is kind of redundant with the one above. I think we updated to 4.3, and then 4.4, in two separate PRs.)